'INT' : redefinition error

I'm getting the following compile error in QT for Windows.
qrsaencryption.h:21: error: C2371: 'INT' : redefinition; different basic types

The INT type is already defined somewhere else:
C:\Program Files (x86)\Windows Kits\8.1\include\shared\minwindef.h:176

It should be  a good idea to guard your code in a namespace.

--- a/src/Qt-RSA/qrsaencryption.cpp
+++ b/src/Qt-RSA/qrsaencryption.cpp
@@ -12,7 +12,7 @@
 #include <ctime>
 #include <chrono>
 
-INT eulerFunc(const INT &p, const INT &q) {
+QRSAEncryption::INT eulerFunc(const QRSAEncryption::INT &p, const QRSAEncryption::INT &q) {
     return (p - 1) * (q - 1);
 }
 
@@ -46,7 +46,7 @@ QRSAEncryption::Rsa QRSAEncryption::getBitsSize(const QByteArray &key) const {
     return QRSAEncryption::Rsa::Invalid;
 }
 
-INT QRSAEncryption::fromArray(const QByteArray &array) const {
+QRSAEncryption::INT QRSAEncryption::fromArray(const QByteArray &array) const {
     INT res = 0;
     res.fromHex(array.toHex().toStdString());
     return res;
@@ -67,7 +67,7 @@ QByteArray QRSAEncryption::toArray(const INT &i, short sizeBlok) {
     return res.left(sizeBlok);
 }
 
-INT QRSAEncryption::randomNumber(bool fullFill) const {
+QRSAEncryption::INT QRSAEncryption::randomNumber(bool fullFill) const {
 
     srand(std::chrono::duration_cast<std::chrono::nanoseconds>
           (std::chrono::system_clock::now().time_since_epoch()).count()
@@ -93,7 +93,7 @@ INT QRSAEncryption::randomNumber(bool fullFill) const {
     return res;
 }
 
-INT QRSAEncryption::toPrime(INT n) const {
+QRSAEncryption::INT QRSAEncryption::toPrime(INT n) const {
     if (!(n % 2)) {
         ++n;
     }
@@ -112,7 +112,7 @@ INT QRSAEncryption::toPrime(INT n) const {
     }
 }
 
-INT QRSAEncryption::randomPrimeNumber(INT no) const {
+QRSAEncryption::INT QRSAEncryption::randomPrimeNumber(INT no) const {
     srand(static_cast<unsigned int>(time(nullptr)));
 
     // max INT
@@ -124,7 +124,7 @@ INT QRSAEncryption::randomPrimeNumber(INT no) const {
     return p;
 }
 
-INT QRSAEncryption::extEuclid(INT a, INT b) const {
+QRSAEncryption::INT QRSAEncryption::extEuclid(INT a, INT b) const {
     INT x = 0, y = 1, u = 1, v = 0, gcd = b, m, n, q, r;
     while (a != 0) {
         q = gcd / a;
@@ -255,7 +255,7 @@ bool QRSAEncryption::generatePairKey(QByteArray &pubKey, QByteArray &privKey) {
             e -= 2;
         } while((!isMutuallyPrime(eilor, e)));
 
-        d = extEuclid(eilor , e);;
+        d = extEuclid(eilor , e);
 
         while(d < 0 ) {
             d += eilor;

--- a/src/Qt-RSA/qrsaencryption.cpp
+++ b/src/Qt-RSA/qrsaencryption.cpp
@@ -12,6 +12,11 @@
 #include <ctime>
 #include <chrono>
 
+#define KEY_GEN_LIMIT 10
+
+const QString SIGN_MARKER = "-SIGN-";
+const int signMarkerLength = SIGN_MARKER.length();
+
 QRSAEncryption::INT eulerFunc(const QRSAEncryption::INT &p, const QRSAEncryption::INT &q) {
     return (p - 1) * (q - 1);
 }

--- a/src/Qt-RSA/qrsaencryption.h
+++ b/src/Qt-RSA/qrsaencryption.h
@@ -18,18 +18,18 @@
 
 #include <bigint.h>
 
-typedef BigInt INT;
 
 #define KEY_GEN_LIMIT 10
 
 static const QString SIGN_MARKER = "-SIGN-";
 static const int signMarkerLength = SIGN_MARKER.length();
 
-typedef QCryptographicHash::Algorithm HashAlgorithm;
 
 class Qt_SECRETSHARED_EXPORT QRSAEncryption
 {
 public:
+    typedef BigInt INT;
+    typedef QCryptographicHash::Algorithm HashAlgorithm;
 
     enum Rsa {
         Invalid = 0,

--- a/src/Qt-RSA/qrsaencryption.h
+++ b/src/Qt-RSA/qrsaencryption.h
@@ -19,12 +19,6 @@
 #include <bigint.h>
 
 
-#define KEY_GEN_LIMIT 10
-
-static const QString SIGN_MARKER = "-SIGN-";
-static const int signMarkerLength = SIGN_MARKER.length();
-
-
 class Qt_SECRETSHARED_EXPORT QRSAEncryption
 {
 public:

